projects/plugins/boost/app/assets/src/js/stores/critical-css-state.ts

```diff
@@ -162,7 +165,7 @@ export const regenerateCriticalCss = async () => {
 	// This will update the store without triggering a save back to the server.
 	cssStateStore.override( freshState );
 
-	const $isCloudCssEnabled = get( isModuleEnabledStore( 'cloud_css' ) ) || false;
+	const $isCloudCssEnabled = get( modulesState ).cloud_css?.active || false;
```

Comment:
This variable shouldn't be prefixed with `$` as `$` has a special meaning for a Svelte Store reference. Note that this is different in `derived` function, where the common practice is `derived( storeName, ( $storeName ) => ` because you're actually getting the store.